lms/views/lti/oidc.py

```diff
@@ -72,7 +72,7 @@ def oidc_view(request):
         "prompt": "none",
         "login_hint": params["login_hint"],
         "lti_message_hint": params["lti_message_hint"],
-        "client_id": params["client_id"],
+        "client_id": registration.client_id,
```

Comment:
:+1: Seems like the better way to do it anyway.

I wonder if it's worth saying something like:

```suggestion
        # We might not have been provided the client id in the request, so use
        # the client id from the registration
        "client_id": registration.client_id,
```